packages/block-library/src/site-tagline/index.php

```diff
@@ -17,11 +17,18 @@ function render_block_core_site_tagline( $attributes ) {
 	if ( ! $site_tagline ) {
 		return;
 	}
+
+	$tag_name           = 'p';
 	$align_class_name   = empty( $attributes['textAlign'] ) ? '' : "has-text-align-{$attributes['textAlign']}";
 	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $align_class_name ) );
 
+	if ( isset( $attributes['level'] ) ) {
+		$tag_name = 0 === $attributes['level'] ? 'p' : 'h' . (int) $attributes['level'];
+	}
```

Comment:
```suggestion
	if ( isset( $attributes['level'] ) && 0 !== $attributes['level'] ) {
		$tag_name = 'h' . (int) $attributes['level'];
	}
```

Nit: Since the initial value of `$tag_name` is `p`, I don't think the ternary operator is needed.